### PR TITLE
Gigasecond

### PR DESCRIPTION
**Pull request by amymhaddad**
@paulghaddad Please review this program and its tests. I have one question about a test that I wrote:

`assert gigasecond(datetime.now()) == datetime.now() + seconds`

I get an error when I run the above line, and I'm not clear why?

### CODE REVIEW

File: gigasecond/test_gigasecond.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+def test_recently_apast_date_time():
+    assert gigasecond(datetime(2011, 4, 25)) == datetime(2011, 4, 25) + seconds
+
+
+def test_very_old_date_time():
+    assert gigasecond(datetime(1886, 2, 11)) == datetime(1886, 2, 11) + seconds
+
+
+# def test_todays_date():
```
Comment (paulghaddad) [inline gigasecond/test_gigasecond.py]: I wouldn't even test this...this is one of the hardest things to test.  The two `.now()` calls will be separated by a millisecond and will always fail. To get this test to work, you'd have to do:

````
current_time = datetime.now() # assures this current time is exactly the same
assert gigasecond(current_time == current_time + seconds
```

File: gigasecond/test_gigasecond.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+#     # import pdb; pdb.set_trace()
+#     assert gigasecond(datetime.now()) == datetime.now() + seconds
+
+
+def test_future_date_time():
+    assert gigasecond(datetime(2020, 4, 11)) == datetime(2020, 4, 11) + seconds
```
Comment (paulghaddad) [inline gigasecond/test_gigasecond.py]: You can extract `datetime(2020, 4, 11)` as a named variable, otherwise it's duplication.

**Review by paulghaddad — CHANGES_REQUESTED** [review]
Nice work! Two small comments to consider.

**Comment by amymhaddad** [comment]
@paulghaddad Ready for final review. Here are my changes:
1. Updated this test:
```
def test_todays_date():
    current_time = datetime.now()
    assert gigasecond(current_time) == current_time + seconds
```
2. Used `birthdate `variable for all tests to reduce duplication of datetime object.

File: gigasecond/test_gigasecond.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+from gigasecond import gigasecond
+from datetime import datetime, timedelta
+
+seconds = timedelta(seconds=1000000000)
```
Comment (paulghaddad) [inline gigasecond/test_gigasecond.py]: Nice job making this a constant. You could even make it all upcase.

**Review by paulghaddad — APPROVED** [review]
Approved! Nice job!

File: gigasecond/test_gigasecond.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+
+def test_recent_date_time():
```
Comment (paulghaddad) [inline gigasecond/test_gigasecond.py]: Nice clean tests with correct style.

**Pull request merged**